package.json

```diff
@@ -15,7 +19,8 @@
     "@types/node": "^20.8.10",
     "@types/react": "^18.2.22",
     "@types/react-dom": "^18.2.7",
-    "dotenv": "^16.3.1"
+    "dotenv": "^16.3.1",
+    "typescript": "^4.5.5",
+    "uglify-js": "^3.14.2"
```

Comment:
I don't think this is being used any where. In which case we can just remove it altogether.